drivers/telescope/lx200fs2.cpp

```diff
@@ -174,6 +214,28 @@ bool LX200FS2::Park()
         return false;
 }
 
+bool LX200FS2::ReadScopeStatus()
+{
+    bool retval = LX200Generic::ReadScopeStatus();
+    // For FS-2 v1.21 owners, stop tracking once Parked.
+    if (retval && 
+        StopAfterParkS[0].s == ISS_ON &&
+        isConnected() && 
+        !isSimulation() &&
+        TrackState == SCOPE_PARKING)
```

Comment:
Could it be that this is no longer SCOPE_PARKING after calling the base class?

Reply:
Simply adding SCOPE_PARKED to this logic messed things up, but maybe I could figure out how to intercept this in a smarter way